ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -87,6 +91,39 @@
       .put("/submit/sslreports/", "1585144") //
       .build();
 
+  private static final String REDACTED_SEARCH_CODE_VALUE = "other.scrubbed";
+
+  // The key format is <provider>.in-content:[sap|sap-follow-on|organic]:[<code>|none]
+  private static final Pattern DESKTOP_SEARCH_COUNTS_PATTERN = Pattern
+      .compile("(.+\\.in-content.[^:]+:)(.*)");
```

Comment:
This could be slightly more precise:
```suggestion
      .compile("([^.]+\\.in-content[:.][^:]+:)(.*)");
```